botorch/acquisition/proximal.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+r"""
+A wrapper around AquisitionFunctions to add proximal weighting of the
+acquisition function. Acquisition function is weighted via a squared exponential
+centered at the last training point, with varying lengthscales corresponding to
+'proximal_weights'. Can only be used with acqusition functions based on single
+batch models.
+
+Small values of proximal_weights corresponds to strong biasing towards recently
```

Comment:
```suggestion
Small values of `proximal_weights` corresponds to strong biasing towards recently
```
just for consistency